app/human-detector-app/screens/GroupScreen.tsx

```diff
@@ -18,6 +19,8 @@ export default function GroupScreen({ navigation }: Props): React.ReactElement {
     throw new Error('Error in GroupScreen.');
   }
 
+  const [, setGroups] = React.useState<Group[]>(userContext.groupList);
```

Comment:
Do we need this? We're never using the value anywhere, and it looks like we're always calling it with `userContext.groupList`. If we do need some state, it can probably be internal to the `DeleteGroupButton` component

Reply:
I tried this out and it wasn't working the same for me. Believe we need this IN GroupScreen as otherwise this screen isn't rerendered when we call setGroups().

Reply:
Doh, didn't realize we needed it for a refresh